node/src/components/consensus/protocols/highway.rs

```diff
@@ -696,6 +701,15 @@ where
         self.highway.deactivate_validator()
     }
 
+    fn set_evidence_only(&mut self) {
+        // TODO: We could also drop the finality detector and round success meter here. Maybe make
+        // HighwayProtocol an enum with an EvidenceOnly variant?
```

Comment:
We probably could, yeah.

Reply:
I created [NDRS-936](https://casperlabs.atlassian.net/browse/NDRS-936) for it.